crates/next-custom-transforms/tests/fixture/source-maps/turbopack/client-graph/production/server-actions/1/output.js

```diff
@@ -0,0 +1,8 @@
+"use turbopack no side effects";
+export { getHeaders } from "data:text/javascript,%2F%2A%20__next_internal_action_entry_do_not_use__%20%5B%7B%22006c24672f968b0713dae406536f7dd6fe7d9a7fa0%22%3A%22getHeaders%22%7D%2C%22%2Fapp%2Fitem.js%22%5D%20%2A%2F%22use%20turbopack%20no%20side%20effects%22%3Bimport%7BcreateServerReference%2CcallServer%2CfindSourceMapURL%7Dfrom%22private-next-rsc-action-client-wrapper%22%3Bexport%20var%20getHeaders%3D%2F%2A%23__PURE__%2A%2FcreateServerReference%28%22006c24672f968b0713dae406536f7dd6fe7d9a7fa0%22%2CcallServer%2Cvoid%200%2CfindSourceMapURL%2C%22getHeaders%22%29%3B";
```

Comment:
No sourceMappingURL in production?

I guess we also want that in production, don't we?

Reply:
No, we don't want it in production, otherwise it would include the server actions source code in the browser source maps:
https://github.com/vercel/next.js/pull/76157

Anyway, I didn't do this explicitly here, it just doesn't emit a sourceMappingURL if there are no source mappings.